src/websocket/websocket_endpoint_base.h

```diff
@@ -69,8 +69,17 @@ namespace sensr {
       con->set_open_handler([this] (websocketpp::connection_hdl hdl) {
         status_ = Status::kOpen;
       });
-      con->set_close_handler([this] (websocketpp::connection_hdl hdl) {
-        status_ = Status::kClosed;
+      con->set_close_handler([this, connection = con] (websocketpp::connection_hdl hdl) {
+        auto close_code = connection->get_remote_close_code();
+        std::string close_reason = connection->get_remote_close_reason();
+        if (close_code == websocketpp::close::status::abnormal_close) {
+          OnFail("SENSR bened the connection because of no responding.");
```

Comment:
"bened" is supposed to be "ended" or "closed", right?

Reply:
`The connection to SENSR was banned because of no responding.`
How about now?